test/bats/tests/gcp/gcp_setup.bash

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+
+# Copyright 2020 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+gcp_project=${GCP_PROJECT_NAME:-"gcp-e2e-test2"}
+sa_account_path="test/bats/tests/gcp/gcpsajson.json"
```

Comment:
This should default to `GCP_SA_JSON`, and also if `test/bats/tests/gcp/gcpsajson.json` is the default, it should also be added to the git-ignore to help avoid accidentally committing/pushing it as this file is sensitive and grants access to the users's GCP account.